samples/camera/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _camera-sample:
+
+Camera Application
```

Comment:
Anticipating we may add other camera modules later, this label and title are a bit too generic.  Better would be:
```
.. _ucam-camera-sample:

uCAM-III Camera Application
###########################
```